docs/markdown/snowplow_web_overview.md

```diff
@@ -616,6 +616,21 @@ Using `event_id` de-duplication as an example, for duplicates we:
 
 The same methodology is applied to `page_view_id`s, however we order by `derived_tstamp`.
 
+## Databricks Specific Information
+
+You can connect to Databricks using either the `dbt-spark` or the `dbt-databricks` connectors. The `dbt-spark` adapter does not allow dbt to take advantage of certain features that are unique to Databricks, which you can take advantage of when using the `dbt-databricks` adapter. Where possible, we would recommend using the `dbt-databricks` adapter.
+
+### Unity Catalog support
+
+With the rollout of Unity Catalog (UC), the `dbt-databricks` adapter has added support in dbt for the three-level-namespace as of `dbt-databricks>=1.1.1`. As a result of this, we have introduced the `snowplow__databricks_catalog` variable which should be used **if** your Databricks environment has UC enabled, and you are using a version of the `dbt-databricks` adapter that supports UC. The default value for this variable is `hive_metastore` which is also the default name of your UC, but this can be changed with the `snowplow__databricks_catalog` variable.
+
+Since there are many different situations, we've created the following table to help guide your setup process (this should help resolve the `Cannot set database in Databricks!` error):
+
+|                                             | Adapter supports UC and UC Enabled                                                                     | Adapter supports UC and UC not enabled         | Adapter does not support UC                                                                           |
```

Comment:
I think due to this logic: `var('snowplow__databricks_catalog', 'hive_metastore') if target.type in ['databricks'] ` the adapter does not support UC where the atomic table is used (top right grid) will still require snowplow__databricks_catalog = 'atomic' to avoid the error message in case it is Databricks < 1.1.1 connector that they are using.

Reply:
There is a section above for databricks unity catalog variable setup starting from line 108, which should be updated to fit this latest logic. Or perhaps just add a pointer to the detailed explanation below which you did here. (very nicely though, really clear instructions!)